src/main.cpp

```diff
@@ -26,13 +26,15 @@
 #include "platform.h"
 #include "version.h"
 
+#include "util/threadpool.hpp"
+
 #ifdef WIN32
 #include "eoserv_windows.h"
 #include "extra/ntservice.hpp"
 #endif // WIN32
 
-volatile std::sig_atomic_t eoserv_sig_abort = false;
-volatile std::sig_atomic_t eoserv_sig_rehash = false;
+extern volatile std::sig_atomic_t eoserv_sig_abort;
+volatile std::sig_atomic_t eoserv_sig_rehash;
 volatile bool eoserv_running = true;
```

Comment:
I think we could specify the variables from 36-38 as static in addition to volatile.

Reply:
Same comment as above, but I think I unintentionally removed the `=false` initializer from `eoserv_sig_rehash`, so I need to add that back in.